great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -94,6 +94,7 @@ def run(
         recompute_existing_parameter_values: bool = False,
         reconciliation_directives: ReconciliationDirectives = DEFAULT_RECONCILATION_DIRECTIVES,
         rule_state: Optional[RuleState] = None,
+        enable_progress_bars: bool = True,
```

Comment:
@cdkini Is this the best way -- as opposed to the mechanism in `DataContext` (if it still exists, I mean).  Thanks!

Reply:
does the `Rule` have a reference to a `DataContext`? I know the `RuleBasedProfiler` does.

My thought here is to expose a configuration option through the `Rule.run()` method so we can pass the RBP's progress_bar value.

Reply:
@cdkini The `Rule` object has access to `DataContext` only through its builders (e.g., `DomainBuilder`) -- I am not sure whether or not this will satisfy your goals.  I do feel that adding additional flags for an inherently UI functionality does not feel right.  In my conversations with @donaldheppner an idea surfaced: what would be really great if we could build our progress bar functionality in such a way that objects can track their own progress internally, and the external progress bar component could query them for the progress status and display it accordingly.  Unfortunately, I could not find a way to do this with `tqdm`, because it is instantiated in a way that wraps the actual `Iterable`, which the progress bar is supposed to track, so they are always entangled.  Happy to discuss.  Thanks.